bin/tx-prover/Cargo.toml

```diff
@@ -47,13 +47,15 @@ figment = { version = "0.10", features = ["toml", "env"] }
 miden-lib = { workspace = true, default-features = false }
 miden-objects = { workspace = true, default-features = false }
 miden-tx = { workspace = true, default-features = false }
+pollster = "0.4"
```

Comment:
Do we still use `pollster` somewhere? (on a quick look I didn't see where it is used).

Reply:
We don't use it anymore.